src/Native/CMakeLists.txt

```diff
@@ -4,6 +4,11 @@ project(CoreRT)
 # Include cmake functions
 include(functions.cmake)
 
+string_has_prefix(${CMAKE_GENERATOR} "Visual Studio 15 2017" GENERATOR_IS_VS2017)
```

Comment:
@janvorli Thank you! For some reason I couldn't find that in the documentation. I will get that fixed later today.